### PR TITLE
docs: login with LinkedIn

### LINKED ISSUE #1247: Create documentation for adding LinkedIn social sign in provider
### Preflight checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions.
- [X] I agree to follow this project's [Code of Conduct](https://github.com/ory/docs/blob/master/CODE_OF_CONDUCT.md).
- [X] I have read and am following this repository's [Contribution Guidelines](https://github.com/ory/docs/blob/master/CONTRIBUTING.md).
- [ ] This issue affects my [Ory Network](https://www.ory.sh/) project.
- [ ] I have joined the [Ory Community Slack](https://slack.ory.sh).
- [ ] I am signed up to the [Ory Security Patch Newsletter](https://ory.us10.list-manage.com/subscribe?u=ffb1a878e4ec6c0ed312a3480&id=f605a41b53).

### Describe your problem

Document this feature:

https://github.com/ory/kratos/pull/3079

### Describe your ideal solution

-

### Workarounds or alternatives

-

### Version

master

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by vinckr**
see linked PR to add to console UI. 

closes #1247

### CODE REVIEW

**Review by fbolton — APPROVED** [review]
Looks good to me, @vinckr !

I noticed that the LinkedIn social sign-in is not yet available in the Ory Console. Do you need to wait before merging the doc update, until after the feature gets release?

**Comment by vinckr** [comment]
Thanks @fbolton 

Yes lets wait until the linked PR is merged at least. 
It already works using the CLI, just the option is not present in the UI yet.

**Comment by tomekpapiernik** [comment]
The PR to add LinkedIn to the console has been open for a while now and there's no end in sight. How about moving the instructions for Ory Console operations to a separate PR so that the CLI path gets merged first?

File: docs/kratos/social-signin/83_linkedin.mdx
Comment (tomekpapiernik) [inline docs/kratos/social-signin/83_linkedin.mdx:22]:
```suggestion
4. [Create a LinkedIn application](https://developer.linkedin.com/), verify it and [activate **Sign In with LinkedInV2**](https://www.linkedin.com/pulse/how-get-signin-linkedin-work-taric-andrade/).
```

Where do you get the `V2` in the name from?

**Comment by vinckr** [comment]
WDYM no end in sight? when the network PR is merged this is done, and that PR just needs some test to pass.
I would just wait until the network PR is merged, then we don't need to do any work here. The users who asked for it already figured it out.

**Comment by jonas-jonas** [comment]
@tomekpapiernik the tests in the network PR is still failing, because the docs page for the provider does not exist yet. So we need to merge this first, to get the network PR green and be able to merge it over there. Would you be fine with merging this PR as is?

File: docs/kratos/social-signin/83_linkedin.mdx
Comment (tomekpapiernik) [inline docs/kratos/social-signin/83_linkedin.mdx:110]: Unnecessary step, please remove it.

File: docs/kratos/social-signin/83_linkedin.mdx
Comment (tomekpapiernik) [inline docs/kratos/social-signin/83_linkedin.mdx:112]:
```suggestion
   ory get identity-config {project-id} --format yaml > identity-config-$project-id.yaml
```

**Review by tomekpapiernik — CHANGES_REQUESTED** [review]

File: docs/kratos/social-signin/83_linkedin.mdx
Comment (vinckr) [reply docs/kratos/social-signin/83_linkedin.mdx:22]: That is what using OIDC is called at LinkedIn. 
They also have a v1 version that is using something different (oauth1) i think

**Comment by tomekpapiernik** [comment]
@vinckr this was opened 4 weeks ago, the last significant sign of life in the other PR is from last week so... :D 
@jonas-jonas I get it. If we're sure it won't take long to add linkedin to the console then I'm okay with that. Let's just get the PR green first, I commented on a few things.

File: docs/kratos/social-signin/83_linkedin.mdx
Comment (tomekpapiernik) [reply docs/kratos/social-signin/83_linkedin.mdx:22]: I don't see a way to create a `V1` app through their dev portal. Adding a version denominator isn't necessary. Please remove it.

Comment (vinckr) [reply docs/kratos/social-signin/83_linkedin.mdx:22]: Sorry, not sure what you are looking at, but it is necessary to have this because there are 2 way to do "login with linkedin" 

v2 will work with Ory, the other one not.

Screenshot 2023-03-20 at 12 29 56

File: docs/kratos/social-signin/83_linkedin.mdx
Comment (vinckr) [reply docs/kratos/social-signin/83_linkedin.mdx:110]: Tbh if you include this you can just copy & paste the code example, if you just have 
`ory get identity-config {project-id} --format yaml > identity-config.yaml`
you need to change the code and you cant just copy&paste it

I change it so it is the same as with the other tutorials, but if you export the project ID its much more practical for everyday use IMO.

**Pull request merged**
